### PR TITLE
[FIX] web: fix the properties field in dark mode

### PR DESCRIPTION
**Pull request by std-odoo**
Purpose
=======
Fix the properties types image that should be light and not dark in dark mode. Fix the background color of the selection options.

Task-3117416

### CODE REVIEW

**Comment by julien-banken** [comment]
Hello @std-odoo,

Thank you for your PR. I don't have much comments about the PR. It's a cool change :smile: May I ask you to also change the text color of the input fields ? :pray: It seems like the color contrast is not high enough on the dark theme: People with disabilities may not be able to easily read the text of the input fields.

In `addons/web/static/src/views/fields/properties/properties_field.scss`,

From:

```css
.o_input,
.o_dropdown_button,
.o_datepicker_button {
    @include print-variable(o-input-border-color, $o-form-lightsecondary);
    color: initial !important;
}
```

To:

```css
.o_input,
.o_dropdown_button,
.o_datepicker_button {
    @include print-variable(o-input-border-color, $o-form-lightsecondary);
}
.o_dropdown_button,
.o_datepicker_button {
    color: initial !important;
}
```

Yeah... It's cosmetic changes and I know we are not suppose to do that in stable but I feel like it's rather important to have text people can read easily.

File: addons/web/static/src/views/fields/properties/property_definition.xml [hunk 1]
```diff
@@ -54,10 +54,10 @@
                             title="state.typeLabel" togglerClass="'btn btn-link d-flex p-0 w-100'">
                             <t t-set-slot="toggler">
                                 <div class="o_input_dropdown w-100 o_field_property_dropdown">
+                                    <img t-attf-src="/web/static/src/views/fields/properties/icons/{{state.propertyDefinition.type}}.png"/>
```
Comment (awa-odoo) [inline addons/web/static/src/views/fields/properties/property_definition.xml:57]: Is it necessary to do this change because of dark mode? Does not seem related.

Not great to change the field templates in stable, even if the change has a low impact.

File: addons/web/static/src/views/fields/properties/property_definition.scss [hunk 2]
```diff
@@ -29,6 +29,10 @@
             height: 20px;
         }
 
+        .o_field_property_dropdown > img {
+            margin-right: -18px;
```
Comment (awa-odoo) [inline addons/web/static/src/views/fields/properties/property_definition.scss:33]: Meh, negative margins ^^

**Review by awa-odoo — COMMENTED** [review]
Hello @std-odoo 

Just a couple questions.

Cheers!

File: addons/web/static/src/views/fields/properties/property_definition.xml (hunk 1 above)
Comment (std-odoo) [reply addons/web/static/src/views/fields/properties/property_definition.xml:57]: yes it's needed, otherwise we can't apply a filter on it (and it's better to fix it that way, rather than duplicating all the images imo)

File: addons/web/static/src/views/fields/properties/property_definition.scss (hunk 2 above)
Comment (std-odoo) [reply addons/web/static/src/views/fields/properties/property_definition.scss:33]: the bottom border it set on the input, and I can't move the image inside of the `<input/>` :/

The only other solution I have in mind is to add CSS to remove the border of the input, and add it on a parent element, but it will require even more code (to change the color on focus, etc)

File: addons/web/static/src/views/fields/properties/property_definition.xml (hunk 1 above)
Comment (awa-odoo) [reply addons/web/static/src/views/fields/properties/property_definition.xml:57]: Ok then, can you just explain in the commit message that we change from background-color to a real image tag so that we can apply the "filter" css function on it in dark mode?

Comment (std-odoo) [reply addons/web/static/src/views/fields/properties/property_definition.xml:57]: @awa-odoo Done :)

**Comment by awa-odoo** [comment]
@robodoo r+
